docs/02-install.md

```diff
@@ -144,6 +146,66 @@ file in your installation artifacts directory and simply re-export it:
 
     export KUBECONFIG=/path/to/something/auth/kubeconfig
 
+# Advnaced Installations
+While the OpenShift 4 installer's purpose in life is to streamline operations
+in order to guarantee success, there are a few options that you can adjust by
+using a configuration file. Namely, you can change the default number of
+instances, and you can change both the master and initial worker EC2 instance
+types.
+
+## Generate Installer Configuration File
+This example will use the `--dir` option. The first step is to ask the
+installer to pre-generate an installer configuration file,
+`install-config.yaml`:
+
+```bash
+openshift-install --dir /path/to/something create install-config
+```
+
+After following the same interactive prompts you saw earlier when performing
+a basic/default installation, this will generate a file called
+`install-config.yaml` in the folder `/path/to/something`. The important
+stanzas in the file to examine are the two `machines` stanzas:
+
+```yaml
+machines:
+- name: master
+  platform: {}
+  replicas: 3
+- name: worker
+  platform: {}
+  replicas: 3
+```
+
+You will notice that the `platform` sections are empty. That's because the
+`platform` is generally defined towards the end of the file along with the
+AWS region.
+
+Let's modify our config file to specify that we want 6 initial workers, all
+of size `c5.xlarge`. Again, the only section to modify is the `machines`
+section:
+
+```yaml
+machines:
+- name: master
+  platform: {}
+  replicas: 3
+- name: worker
+  platform: 
+    aws:
+      type: c5.xlarge
```

Comment:
Do we need to hold this until 0.10.1 gets more traction?  I think it was broken in 0.10.0.

Reply:
> Are you saying this instance type/size is broken, or the whole usage of this?

Before 0.10.1, the installer would silently ignore `type` and `rootVolume` settings from `install-config.yaml` when creating machine sets.  That got fixed in 0.10.1.

Reply:
I didn't think there was going to be an "official" update during the beta... but the try.openshift.com workflow still sends you to GH to download the installer release... so people are likely to have 0.10.1 or be able to get it?

Should I include a note about 0.10.1?

Reply:
> ... so people are likely to have 0.10.1 or be able to get it?

Yes.

> Should I include a note about 0.10.1?

That depends on the audience.  Tushar was supposed to be give HTB folks some guidance on which installer releases were part of the program, but I dunno if/where that happened.  Mentioning 0.10.1 would be conservative, but maybe folks have already dropped 0.10.0 and earlier.  And soon all of this will be ancient history anyway.

Reply:
I'll take a gamble and leave 0.10.1 out of the picture and we'll see. We talked about changing this on a recent beta call and no one has complained that it isn't working -- yet.